.vex/v0.41.0.vex.json

```diff
@@ -2,21 +2,35 @@
   "@context": "https://openvex.dev/ns/v0.2.0",
   "@id": "https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.41.0/v0.41.0.vex.json",
   "author": "Inspektor Gadget Security Team <security@inspektor-gadget.io>",
-  "timestamp": "2025-08-21T08:52:32.416055456+02:00",
-  "version": 1,
+  "version": 2,
   "statements": [
     {
       "vulnerability": {
         "name": "CVE-2025-54388"
       },
-      "timestamp": "2025-08-21T08:52:32.416057292+02:00",
       "products": [
         {
           "@id": "pkg:github/inspektor-gadget/inspektor-gadget@v0.41.0"
         }
       ],
       "status": "not_affected",
-      "justification": "vulnerable_code_not_in_execute_path"
+      "justification": "vulnerable_code_not_in_execute_path",
+      "timestamp": "2025-08-21T06:52:32.416057292Z"
```

Comment:
It's confusing that the timestamp is the same as before but in a different timezone. Did you specify the timezone when using the `vexctl` command?